imports/plugins/core/ui-navbar/client/components/navbar/navbar.js

```diff
@@ -73,6 +73,14 @@ Template.CoreNavigationBar.helpers({
       }
     };
   },
+
+  staticPagesMenu() {
```

Comment:
I'm curious to know why this function is documented.

Reply:
I did not get you. do you mean why it is not documented?

Reply:
Yes, I do.